### DISCUSSION TITLE
#68 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0x-stan** [tog-ETH-er/togETHer#53] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/gtu-mubt-wom

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

5个PR #977 ～ #982

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会
 
- 《Zuzalu的游民生活》by kelly
   - Youtube: https://www.youtube.com/watch?v=ttx44ziQUqA
   - Bilibili: https://www.bilibili.com/video/BV1qm4y1s7Kk/

- 《LVR解读--AMM LP做市风险分析》by HP
   - Youtube: https://www.youtube.com/watch?v=PlwUU3Xr1-g
  - Bilibili: https://www.bilibili.com/video/BV1EM4y1W7ML

## Other

- Hacker House Tokyo

**Comment by luckyyang** [comment]
Rebase Community 
 
- web3 极客日报: 最近 14 期 http://daily.rebase.network/ 
- 招聘: 3 个团队发布5个招聘 https://github.com/rebase-network/who-is-hiring/issues 
- web3极客公开课（web3geekforge ）2期:  
  - https://space.bilibili.com/382886213/video 
- 2 个活动 https://github.com/rebase-network/event-pool/issues
- 武汉hacker house招募完成

**Comment by Stamp9** [comment]
### 原语里弄

研究课题进行中。

**Comment by Thegaram** [comment]
#### Crypto Tech Night

No new update this week.
